spec/models/post_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  let(:first_user) { User.new(name: 'Tom', photo: nil, bio: nil, posts_counter: 0) }
+  let(:first_post) { first_user.posts.create(title: 'Post 1', text: 'Hello', comments_counter: 0, likes_counter: 0) }
+
+  before { first_user.save }
+
+  context 'validations' do
+    it 'comments_counter should not be less than 0' do
+      first_post = Post.new(author: first_user, title: 'Hello', text: 'This is my first post')
+      first_post.comments_counter = -3
+      expect(first_post).to_not be_valid
+    end
+
+    it 'comments_counter should be an integer' do
+      first_post = Post.new(author: first_user, title: 'Hello', text: 'This is my first post')
+      first_post.comments_counter = 'one'
+      expect(first_post).to_not be_valid
+    end
+
```

Comment:
[ Required ]
- [ ] Just as the previous reviewer mentioned, please add a test for the post title to be present. It's required that a post title must not be blank. Kindly add a test to validate that 👍🏽

[ Optional Change ]
I am marking this change as optional because the previous reviewer did not point it but I highly recommend your work on it.

- [ ] You're expected to have a test for the Title of the post to not exceed 250 characters. Kindly consider working on that 👍🏽

<img width="640" alt="Screenshot 2023-08-18 at 8 23 11 PM" src="https://github.com/WilhelmK109/BlogApp/assets/50941074/0b64e623-bef6-4382-9dd6-b5b8ce10d109">